source/tutorials/openmm.rst

```diff
@@ -17,4 +23,128 @@ folder of the GitHub repository. It contains:
   NanoVer simulation using a python client.
 * `openmm_neuraminidase`: A notebook that demonstrates how to set up an interactive
   OpenMM simulation from scratch with NanoVer and change the visualisations of
-  different atom selections for a protein-ligand system.
\ No newline at end of file
+  different atom selections for a protein-ligand system.
+
+XML Files
```

Comment:
I think maybe this heading ("XML Files") could be refactored to something that makes it clear that these are NanoVer simulation files of an OpenMM simulation, rather than being a general XML file. It could also be good to add a sentence below this heading to explain this.

Reply:
Perhaps something like "NanoVer OpenMM input files"? XML is just a general nested file structure, not specific to NanoVer or OpenMM, and the important thing is that the input files described in this section are specific to OpenMM simulations performed using NanoVer

Reply:
The XML format we are using does not seem to be different from the one OpenMM defines in their documentation. As far as I know, it is not proprietary to NanoVer (The same Serializer Class exists in Openmm with the same XML structure [here](http://docs.openmm.org/latest/api-python/generated/openmm.openmm.XmlSerializer.html?highlight=serializer#openmm.openmm.XmlSerializer) )

Reply:
So there is a slight difference between the OpenMM format and the NanoVer format. NanoVer input files for OpenMM simulations contain a pdb and a number of xml serialized objects. Although the serialized objects are identical to those used by OpenMM (they are actually created used OpenMM), the overall contents and structure of the files is specific to NanoVer and cannot be read by OpenMM on it's own. Explaining this could prevent users misunderstanding the NanoVer input file format.

Reply:
Thanks for clearing this out @rhoslynroebuck. I agree that specifying the overall contents and structure of the XML files in the docs would prevent users from misinterpreting or misusing them.

Reply:
Resolved in commit [f62ca6e](https://github.com/IRL2/nanover-docs/pull/115/commits/f62ca6e5fd4d7a15e0d7843f064081e5d054ccf2)